File: MotivApp/app/src/main/java/com/motiv/piotr/UsersListFragmentContract.java
package com.motiv.piotr;

import androidx.fragment.app.*;
import dagger.*;
import dagger.android.*;
import dagger.android.support.*;
import javax.inject.*;

public interface UsersListFragmentContract {

    public interface View {

        public void usersListAdaptersetData(java.util.List<com.motiv.piotr.User> arg0);

        public void navigationControllerstartUserDetailActivity(com.motiv.piotr.User arg0);
    }

    public interface Presenter {

        public void goRestApigetUsersList();

        public void eloonItemClick(int argument0, com.motiv.piotr.User argument1);
    }
}

File: MotivApp/app/src/main/java/com/motiv/piotr/UsersListFragmentContract.kt
 
package com.motiv.piotr
import kotlin.collections.List

public interface UsersListFragmentContract {

    public interface View {

        public fun usersListAdaptersetData(arg0: List<com.motiv.piotr.User>): Unit
        public fun navigationControllerstartUserDetailActivity(arg0: com.motiv.piotr.User): Unit
    }

    public interface Presenter {

        public fun goRestApigetUsersList(): Unit
        public fun eloonItemClick(argument0: Int, argument1: com.motiv.piotr.User): Unit
    }
}
